src/commands/protocols/publish-paranet/sender/publish-paranet-validate-asset.command.js

```diff
@@ -27,6 +27,66 @@ class PublishParanetValidateAssetCommand extends ValidateAssetCommand {
             storeType = LOCAL_STORE_TYPES.TRIPLE,
         } = command.data;
 
+        // Validate node is in paranet
+        const paranetId = this.paranetService.constructParanetId(blockchain, contract, tokenId);
+        const nodesAccessPolicy = await this.blockchainModuleManager.getNodesAccessPolicy(
+            blockchain,
+            paranetId,
+        );
+        if (nodesAccessPolicy === 1) {
+            const identityId = await this.blockchainModuleManager.getIdentityId(blockchain);
+            const isCuratedNode = await this.blockchainModuleManager.isCuratedNode(
+                blockchain,
+                paranetId,
+                identityId,
+            );
+            if (!isCuratedNode) {
+                await this.handleError(
+                    operationId,
+                    blockchain,
+                    `node with identity id ${identityId} is not a curated node in paranet with paranetid ${paranetId}`,
+                    this.errorType,
+                    true,
+                );
+                return Command.empty();
+            }
+        }
+
+        // Validate asset is in paranet
+        const knowledgeAssetId = await this.paranetService.constructKnowledgeAssetId(
+            blockchain,
+            contract,
+            tokenId,
+        );
+        const isParanetKnowledgeAsset = await this.blockchainModuleManager.isParanetKnowledgeAsset(
```

Comment:
This is not complete validation , it only checks if the asset is part of some paranet. We want to check if it is part of the paranet to which member nodes asset is being published to. 
We should use  getKnowledgeAssetObject function from ParanetKnowledgeAssetsRegistry contract. That object has paranetId field which should match paranetId we are publishing to.

Reply:
Now I see you did this on line 75, my bad, let's just keep the later validation than (one one line 75)